ding/torch_utils/network/soft_argmax.py

```diff
@@ -4,39 +4,36 @@
 
 
 class SoftArgmax(nn.Module):
-    r"""
+    """
     Overview:
-        An nn.Module that computes SoftArgmax
+        An nn.Module that computes the SoftArgmax operation.
```

Comment:
Maybe a description that this softmax is 2-dimension softmax is better.

Reply:
polished now